docs/glossary.md

```diff
@@ -115,11 +131,28 @@ Tickets that reached the end of their window without being called to vote - thes
 
 A mechanism for obtaining free (testnet) coins.
 
+#### Full node
+
+An instance of the Decred daemon `dcrd` which is serving the full version of the blockchain to peers, and fully verifying all of the consensus rules of Decred. 
+
+
+## G
+
+---
+
+#### Genesis block
+
+The very first block of the Decred blockchain, created on the 8th of February 2016.
+
 
 ## H
 
 ---
 
+#### Hardware wallet
+
+A dedicated hardware device which securely stores a wallets private keys. Examples include [Trezor](https://trezor.io/) and [Ledger](https://www.ledger.com/).
```

Comment:
Not all hardware wallets are necessarily all that secure.  I'm not sure that claim should be made here.

Reply:
Removed the word "secure"